styles.css

```diff
@@ -0,0 +1,550 @@
+  /* ---------Header section----------- */
+  .menu_mobile {
+    display: none;
+  }
+
+  .logo {
+    margin-left: 5%;
+  }
+
+  .menu_desktop {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 5%;
+    background: #fff;
+    position: fixed;
+    width: 100%;
+  }
+
+  .nav_menu {
+    list-style: none;
+    display: flex;
+    margin-right: 180px;
+    padding: 10px 12px;
+    gap: 30px;
+  }
+
+  .nav_menu li a {
+    text-decoration: none;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  /* -------Home section--------- */
+  .home {
+    background-color: #ffff;
+    border-bottom-left-radius: 80px;
+    background-image: url("images/headerbg.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position-y: 77px;
+    padding-bottom: 2%;
+  }
+
+  .home-content {
+    margin-left: 10%;
+    padding-top: 12%;
+  }
+
+  .lets-connect-message {
+    margin-left: 50px;
+  }
+
+  .home-content,
+  .my-intro {
+    width: 81%;
+    align-self: flex-start;
+  }
+
+  .connect-media {
+    margin-left: 13px;
+  }
+
+  /* --------Work section---------- */
+
+  .flex {
+    display: flex;
+    flex-direction: row;
+    gap: 48px;
+    border-radius: 20px;
+    border: 2px solid #ffff;
+    margin: 0 5%;
+    background-color: #ffff;
+  }
+
+  .set_top_margin {
+    margin-top: 4%;
+  }
+
+  .set_bottom_margin {
+    margin-bottom: 4%;
+  }
+
+  .flex_content {
+    margin-top: -174px;
+  }
+
+  .mobile_image {
+    display: none;
+  }
+
+  .desktop_image {
+    display: block;
+  }
+
+  .card-info {
+    margin-top: 20px;
+  }
+
+  .reverse {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .canopy-list li {
+    margin-top: -2px;
+  }
+
+  .second li {
+    block-size: fit-content;
+    width: fit-content;
+  }
+
+  /* -----------About section------------ */
+  .about {
+    display: flex;
+    flex-direction: row;
+    gap: 25px;
+    padding: 50px;
+    border-top-right-radius: 80px;
+  }
+
+  .about-content {
+    width: 40%;
+  }
+
+  .about-list {
+    width: 60%;
+    gap: 12px;
+  }
+
+  .languages {
+    display: flex;
+    flex-direction: row;
+    gap: 16px;
+  }
+
+  .add_padding {
+    padding-top: 87px;
+  }
+
+  .about-content .my-intro {
+    width: 100%;
+    margin-left: 1px;
+    margin-top: -3px;
+  }
+
+  .languages li {
+    list-style: none;
+    background-color: #f7f7f9;
+    margin-bottom: 14px;
+    width: 98px;
+    height: 111px;
+    border-radius: 8px;
+  }
+
+  /* -------------Contact Form------------------ */
+  .footer_content {
+    width: 50%;
+    margin: 0 25%;
+    background: none;
+  }
+
+  form input,
+  textarea {
+    border-radius: 0;
+  }
+
+  .submit {
+    align-self: center;
+  }
+
+  .contact_section {
+    background-image: url(images/contact_bg.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
```

Comment:
You have a vertical overflow issue in this section. Please fix it.

<img width="1440" alt="Screen Shot 2022-07-08 at 8 26 38" src="https://user-images.githubusercontent.com/82426682/178001044-482af022-5817-4185-9419-d07a780dbd51.png">